common/logger.go

```diff
@@ -52,6 +52,7 @@ type ILoggerResetable interface {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+// TODO : Can this be deleted?
```

Comment:
this isnt used anywhere can we delete?

Reply:
you mean this method "NewAppLogger" ?

Reply:
I kind of dont want to delete again since it is publicly exported

Reply:
A quick search for NewAppLogger in the Blobfuse2 codebase indicates that it isn't used so I will delete